[DOC] Add new section on Kafka Bridge (#51)
rh-messaging/amq-streams-docs: books/proc-downloading-kafka-bridge.adoc

// Module included in the following assemblies:
//
// assembly-getting-started.adoc

[id='proc-downloading-kafka-bridge-{context}']

> = Downloading an AMQ Streams Archive

Reviewer: This sounds a bit strange since using the _Archive_ alone gives me the sense that it's old. Should we say _ZIP archive_?

```suggestion
= Downloading an AMQ Streams ZIP Archive
```

> An archived distribution of AMQ Streams is available for download from the Red Hat website.

Reviewer: ```suggestion
A zipped distribution of AMQ Streams is available for download from the Red Hat website.
```
